### DISCUSSION TITLE
Camera on 5m pro not visable with Orca Flashforge Slicer

### DISCUSSION THREAD

**Discussion by Plasticsman** [OrcaSlicer/OrcaSlicer#5594] [General]
I just started with the Orca slicer for FF 5m pro.  Able to upload prints by WIFI but can't figure out how to access the 5m pro camera through the slicer. Any thoughts?

Ron

**Comment by fargodwe** [comment]
I know this is a year old thread however i just added a camera to my FF 5M tonight and had the same problem.  If anyone comes to this thread with the same issue remember to do what I forgot:  on the printer go to settings then tap the little round camera icon on the top row then turn on video and pictures.  I feel stupid for forgetting to do it!

**Comment by snuglinux** [comment]
The stream from the camera is conducted at: http://ip-printer:8080/?action=stream, but there is no support in the orca slicer :(
